.changeset/tricky-ducks-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
+---
+
+feat(BottomSheet): add `zindex` prop & improve focus management logic
```

Comment:
looks good, let's attribute @archie252000 for the initial draft.

Reply:
Done.